airflow/sensors/base.py

```diff
@@ -114,15 +118,33 @@ def _orig_start_date(
     )
 
 
+class SkipPolicy(str, enum.Enum):
+    """Class with sensor's skip policies."""
+
+    # if poke method raise an exception, sensor will not be skipped on.
+    NONE = "none"
+
+    # If poke method raises an exception, sensor will be skipped on.
+    SKIP_ON_ANY_ERROR = "skip_on_any_error"
+
+    # If poke method raises AirflowSensorTimeout, AirflowTaskTimeout, AirflowFailException
+    # sensor will be skipped on.
+    SKIP_ON_SOFT_ERROR = "skip_on_soft_error"
+
+    # If poke method raises an exception  different from AirflowSensorTimeout, AirflowTaskTimeout,
+    # AirflowSkipException, sensor will ignore exception and re-poke until timeout.
+    IGNORE_ERROR = "ignore_error"
```

Comment:
I'm wondering whether we could reword this. It looks similar to `SKIP_ON_ANY_ERROR` at the first glance 🤔

Reply:
this is a copy past of the definition of `silent_fail` , could you please make a suggestion , thanks